jerry-libc/jerry-libc.c

```diff
@@ -176,7 +176,7 @@ CALL_PRAGMA (GCC diagnostic pop)
 
 /** Compare two strings. return an integer less than, equal to, or greater than zero
      if s1 is found, respectively, to be less than, to match, or be greater than s2.  */
-int
+int __attr_used___
 strcmp (const char *s1, const char *s2)
```

Comment:
Perhaps we should land this as a different patch, since it is not related to documentation.